drivers/sensor/hx711/Kconfig

```
@@ -0,0 +1,93 @@
+# HX711 Weight Scale interface Sensor configuration options
+# Copyright (c) 2020, George Gkinis
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig HX711
+	bool "HX711 Weight Scale Sensor"
+	depends on GPIO
+	help
+	  Enable driver for the HX711 Weight Scale Sensor.
+
+if HX711
+
+config HX711_OFFSET
+	int "Initial Offset"
+	default 0
+	help
+	  Sensor offset.
+
+config HX711_SLOPE_INTEGER
+	int "Initial Slope. Integer part."
+	default 1
+	help
+	  Sensor slope, integer part.
+
+config HX711_SLOPE_DECIMAL
+	int "Initial Slope. Decimal part."
+	default 0
+	help
+	  Sensor slope, decimal part.
+	  Multiply by 100000. For .51 enter 510000
+
+config HX711_CALIBRATION_WEIGHT_INTEGER
+	int "Calibration weight."
+	default 0
+	help
+	  Known weight for calibration purposes, integer part.
+
+config HX711_CALIBRATION_WEIGHT_DECIMAL
+	int "Calibration weight."
+	default 0
+	help
+	  Known weight for calibration purposes, decimal part.
+	  Multiply by 100000. For .51 enter 510000
+
+config HX711_SAMPLE_FETCH_TIMEOUT_MS
+	int "Timeout for waiting until reading is ready."
+	default 700
+	help
+	  Fill in the time needed for the HX711 module to be ready after Power Up.
+
+choice
+	prompt "HX711 sampling rate"
+	default HX711_RATE_10HZ
+	help
+	  Select sampling rate.
+config HX711_RATE_10HZ
+	bool "10Hz"
+config HX711_RATE_80HZ
```

please add empty line for readability.
```suggestion

config HX711_RATE_80HZ
```

---

Done.